docs/courses/csintro1/intro/makecode-orientation.md

```diff
@@ -33,48 +33,48 @@ https://youtu.be/iEQa-Wd3MRs
 
 [Alternative Video Location](https://aka.ms/40544a-orientation2)
 
-## Student Task #2: Starting your own game
+## Student Task #2: Starting your own game #task-2
 
 1. Press the "Home" button in the upper left corner to return to the homepage
 2. Press the "New Project" button
 3. Find ``||variables:set mySprite to||`` in ``||sprites:Sprites||``, and drag it into ``||loops:on start||``
 4. Make the new ``||sprites:Sprite||`` ``||sprites:say||`` something, by dragging ``||sprites:mySprite say||`` and placing it after ``||variables:set mySprite to||``
 
     ![Sprite Say](/static/courses/csintro1/intro/sprite-say.gif)
-
+<br/>
 5. At the bottom of the screen, select a name to save this game as (for example, "first game"), and click "Download" to download it
 
     ![Downloading the game](/static/courses/csintro1/intro/download.gif)
-
+<br/>
 6. If you are in a class, you may need to turn in this file today, so be sure to keep track of it!
 
-## Student Task #3: Importing your game
+## Student Task #3: Importing your game #task-3
 
 To make sure you saved your project properly, we should import it back into @boardname@!
 
 1. Press the "Home" button in the upper left corner to return to the homepage
 2. The "My Projects" section should now include the game you created in the last task. To the right of it, there should be a button labeled "Import" - press that button
 3. Select "Import File..."
-4. Select the file you saved in task #2
+4. Select the file you saved in [task #2](#task-2)
 5. Your code should now appear in the browser window
 
 Alternatively, you can simply drag the image that you downloaded onto the home page and it will load automatically.
 
-## Student Task #4: Sharing your game
+## Student Task #4: Sharing your game #task-4
 
 There is one other common way that we will keep track of our games: by making links to directly access the code.
 
 1. Press the "Home" button in the upper left corner to return to the homepage
-2. Select your game from task #2 under "My Projects"
+2. Select your game from [task #2](#task-2) under "My Projects"
 3. Select the share button in the upper left corner, as shown below. Verify that you want to publish the code, and keep track of the URL it shows - that is a link directly to your code!
 
 ![Sharing the game](/static/courses/csintro1/intro/sharing.gif)
 
 ## What did we learn?
 
-1. What did the ``||sprites:say||`` block do when you added it to the game in task #2? 
+1. What did the ``||sprites:say||`` block do when you added it to the game in [task #2](#task-2)? 
 2. Compare and contrast the two different ways that we saved our game; what are the benefits of each choice? Which do you think you would use if you wanted to email a game you made to your friend, and why would you make that decision?
-3. Include the image created in task #2.
-4. Include the link created in task #4.
+3. Include the image created in [task #2](#task-2).
+4. Include the link created in [task #4](#task-4).
```

Comment:
I like the anchor tags throughout, I can go through and add them in throughout the rest if you want